src/components/Button/Button.tsx

```diff
@@ -19,26 +20,28 @@ export const Button = ({
   size = 'medium',
   type = 'button',
   colour = 'grey',
+  hover = true,
 }: Props): ReactElement => {
   return (
     <button
       className={cn(
-        className,
-        'font-poppins text-[#333F4C]',
-        'rounded-full transition duration-500 hover:border-[#333F4C] hover:bg-[#333F4C] hover:text-white',
-        'hover:cursor-pointer',
+        'font-poppins',
+        'rounded-full transition duration-500',
         {
-          'py-1 px-5 text-[14px] ': size === 'small',
-          'py-2 px-5 text-[18px]': size === 'medium',
-          'py-3 px-12 text-[22px] font-bold': size === 'large',
-          'bg-transparent': colour === 'clear',
-          'border-2 border-solid border-[#333F4C] bg-[white]': colour === 'white',
-          'border-2 border-solid border-[#A2AAB6] bg-[#A2AAB6]': colour === 'grey',
-          'border-2 border-solid border-[#FFC600] bg-[#FFC600]': colour === 'yellow',
-          'border-2 border-solid border-[#f76d6d] bg-[#f76d6d]': colour === 'red',
-          'border-2 border-solid border-[#A2AAB6] bg-transparent text-inherit': colour === 'clear-border',
-          'cursor-pointer bg-[#5A6675]': active,
-        }
+          'py-1 px-3 text-sm': size === 'small',
+          'py-2 px-6': size === 'medium',
```

Comment:
this component is getting a bit mad. will need to clean this up at some point, we'll leave it for now though